packages/core/src/Page/index.js

```diff
@@ -141,11 +140,8 @@ class Page {
     if (this.siteConfig.titleSuffix) {
       title = (title ? title + TITLE_SUFFIX_SEPARATOR : '') + this.siteConfig.titleSuffix;
     }
-    // construct temporary asset object with only POSIX-style paths
-    const asset = {};
-    Object.entries(this.asset).forEach(([key, value]) => {
-      asset[key] = _.isString(value) ? fsUtil.ensurePosix(value) : value;
-    });
+
+    const asset = { ...this.asset };
```

Comment:
done!

> If we don't need to make a copy anymore

It's being mutated at the moment (line 496), think it'd be best to keep the current behaviour of copying for now. Not sure if it would affect things across multiple runs in live preview for example.